WordPress/gutenberg
# Mobile - HTML View - Add a top margin for the title

- `Gutenberg Mobile PR` -> https://github.com/wordpress-mobile/gutenberg-mobile/pull/4767

Fixes https://github.com/wordpress-mobile/gutenberg-mobile/issues/4351

## What?
Fixes an issue where the notice covers the title in the HTML editor mode.

## Why?
The notice shouldn't cover the title of the post.

## How?
By adding a top margin in the HTML editor to prevent the title from being covered by the notice.

## Testing Instructions
- Open the editor
- Add a title
- Toggle to HTML mode
- **Expect** the notice **to not** cover the title

## Screenshots or screencast 

Before | After
-|-
<kbd><img src="https://user-images.githubusercontent.com/4885740/163785000-3f82242f-ac6f-48ea-9f6d-af8aefe22380.gif" width="250" /></kbd> | <kbd><img src="https://user-images.githubusercontent.com/4885740/163784983-0f36bec1-f920-434e-b250-5194876599f6.gif" width="250" /></kbd>
